tmt/steps/__init__.py

```diff
@@ -110,13 +113,8 @@ class StepData(tmt.utils.NormalizeKeysMixin, tmt.utils.SerializableContainer):
     order: int = tmt.utils.DEFAULT_PLUGIN_ORDER
     summary: Optional[str] = None
 
-    def to_raw(self) -> _RawStepData:
-        """
-        Serialize step data instance to a raw representation.
-
-        The returned value can be used to recreate step data when given
-        to :py:meth:`from_raw`.
-        """
+    def to_spec(self) -> _RawStepData:  # type: ignore[override]
+        """ Convert from a specification a file or CLI option """
```

Comment:
This is not *from* but *to* spec, right?

Reply:
Right, it's been reversed.